internal/controllers/clusterextension_controller_test.go

```diff
@@ -608,57 +608,6 @@ func TestClusterExtensionExpectedBundleDeployment(t *testing.T) {
 	require.NoError(t, cl.DeleteAllOf(ctx, &rukpakv1alpha2.BundleDeployment{}))
 }
 
-func TestClusterExtensionDuplicatePackage(t *testing.T) {
```

Comment:
Fails because we no longer check uniqueness. Do we want to deal with it in some other way without a solver? E.g. iterating over all extensions and checking their packages on each reconcile?